components/merkledb/src/backends/temporarydb.rs

```diff
@@ -55,13 +55,15 @@ impl TemporaryDB {
         // For some reason, using a `WriteBatch` is significantly faster than using `DB::drop_cf`,
         // both in debug and release modes.
         let mut batch = WriteBatch::default();
+        let db = self.inner.rocksdb();
```

Comment:
Nit: IMO, it could make `get_lock_guard` have `pub(super)` visibility and use it here directly.